standalone-tools/wss-scan.sh

```diff
@@ -0,0 +1,121 @@
+###### Information ############################################################################
+# Name:          wss-scan.sh
+# Maintainer:    ODFE Infra Team
+# Language:      Shell
+#
+# About:         This script is to scan the odfe distros for vulnerabilities and licenses
+#                It will scan the repositories and send the WhiteSource link to the mail 
+#                of the user. 
+#
+# Prerequisites: Need to set the Java Path
+#                Need to set the recepient mail in wss-scan.config
+#
+# Usage:         ./wss-scan.sh
+#
+# Starting Date: 10-13-2020
+# Modified Date: 10-15-2020
+###############################################################################################
+
+#Download the WhiteSource Agent 
+curl -LJO -sS https://github.com/whitesource/unified-agent-distribution/releases/latest/download/wss-unified-agent.jar > /dev/null
+
+echo "Download completed"
+
+#Scanning the config file for the user configurations
+source wss-scan.config
+gitRepos=${gitRepos//,/$'\n'}  # change the semicolons to white space
+
+basepath=$baseDirPath"/repos"
+
+if [[ ! -e $basepath ]]; then
+    mkdir -p $basepath
+fi
+ 
+
+echo "Cleaning up scan directories if already present"
+rm -rf $basepath/*
+
+
+#Cloning the desired Repos for scanning 
+for repo in $gitRepos
+do
+        echo "Cloning repo "$gitBasePath$repo
+    git clone "$gitBasePath$repo".git $basepath"/"$repo
+done
+
+cp /dev/null info.txt
```

Comment:
Why?

Reply:
To prepare the file for a second run.

Reply:
Then why not `echo -n > info.txt`.

Reply:
I have to admit this is my first time seeing ` cp /dev/null info.txt` as a method to clean up the file. 😄